### ISSUE TITLE
Footstep sounds for running and walking appear to be swapped

### ISSUE THREAD

**Issue by Nexusuxen** [goonstation/goonstation#13562]
### Labels

[BUG]

### Description

If your speed is set to Run, you hear a footstep sound every half-second to second or so, but if you're walking it sounds like you're running since the footsteps come much more quickly.

### Steps to reproduce

1. Move around with your speed set to Run and notice the slower footstep sounds
2. Set speed to Walk
3. Move around. Notice that the footsteps now happen much more quickly.

### Expected Behavior

For Running to have faster footstep noises and Walking to have slower ones.

### Additional Information & Screenshots

I have no idea how long this has been in the game but I feel like at least a year, probably more, as I definitely recall this behavior occurring ages ago - just never bothered to report it until now! Seems like this was brought up in Issue 2196, but either nothing ever came of it, or the fix broke at some point.

**Comment by frawhst** [comment]
Yeah I made that original issue because it seemed dumb but the code seems to very specifically check if you were walking and play footstep sounds faster if so, only for shoes that `step_lots` (clown shoes and :
https://github.com/goonstation/goonstation/blob/96a81d6e51db7fd1bb1041eec60d2480ac4df88d/code/mob/living/carbon/human.dm#L3172-L3181
(Essentially, for shoes that step a lot, play a 50-volume step every 3 steps when we're running, play a 20 volume step every step if we're walking).
Unclear what the purpose of this is but it's dumb imo and I'd be happy to swap them.
